deck/models.py

```diff
@@ -195,7 +195,9 @@ class Proposal(Activity):
         _('speakerdeck.com/'), max_length=250, null=True, blank=True)
 
     def get_full_slides_url(self):
-        return '{0}{1}'.format('http://www.speakerdeck.com/', self.slides_url)
+        if self.slides_url is not None:
```

Comment:
This can be something like, it's easier to read than the other version:

```
def get_full_slides_url(self):
    if self.slides_url:
        return 'http://www.speakerdeck.com/{0}'.format(self.slides_url)
```

Reply:
Hello @victorfarruda, In a good review process when you did the changes that someone asks for, its nice for you to reply me with you commit, that way i will know exactly the changes that you made.

Thanks <3